### PR TITLE
✨  Add wyrmling

### PR DESCRIPTION
**Pull request by youenPlusquellec**

### CODE REVIEW

File: Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    public class WyrmlingFireball : MonoBehaviour
+    {
+        public Enemy associatedEnemy;
+
+        public void OnTriggerEnter(Collider collider)
+        {
+            if (collider.tag == "Player")
+            {
+                associatedEnemy.Attack(collider.gameObject.GetComponent<Entity>());
+                Debug.Log(collider.gameObject.GetComponent<Entity>().currentHealth);
```
Comment (DoBrcd) [inline Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs:16]:
```suggestion
```

Comment (youenPlusquellec) [inline Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs:16]: EH OH C'EST QUOI CA ?

Comment (wilfried7) [inline Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs:16]: C'EST SCANDALEUX !!!!!!!!!!!!

HONTEUX

**Review by DoBrcd — CHANGES_REQUESTED** [review]

File: Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs (hunk 1 above)
Comment (Ailurus13) [inline Assets/Modules/Enemy/Scripts/WyrmlingFireball.cs:16]: Y'a un Debug Log idiot

**Review by wilfried7 — CHANGES_REQUESTED** [review]
HONTE

**Review by DoBrcd — APPROVED** [review]

**Review by wilfried7 — APPROVED** [review]

**Comment by youenPlusquellec** [comment]
[Prefab Dragonnet de base](https://trello.com/c/NujXhrcn/18-prefab-dragonnet-de-base)

**Pull request merged**